quick/README

```diff
@@ -0,0 +1,2 @@
+archived folder of the 2020 version when the qr code generator was merely used
+for faster input of guest contact data, instead of actual tracking code.
```

Comment:
In that case, I will tag commit 4bbe15f as the 2020 version, in order to make past year's exclusive work easier to find again. Not sure whether it will survive the pull request though.

Reply:
The old state is preserved in the history. I can add a tag to the 2020 version in the repo for reference. I don't think you can add new tags through a PR.

Reply:
Tag added: https://github.com/snue/stuss-tracking/releases/tag/2020_version